specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json

```diff
@@ -594,6 +594,80 @@
         }
       }
     },
+    "/providers/Microsoft.Capacity/reservationOrders/{reservationOrderId}/reservations/{reservationId}/archive": {
+      "post": {
+        "summary": "Archive a `Reservation`.",
+        "description": "Archiving a `Reservation` moves it to `Archived` state.",
+        "operationId": "Reservation_Archive",
+        "x-ms-examples": {
+          "Archive": {
+            "$ref": "./examples/Archive.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ReservationOrderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ReservationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "Reservation, Archive"
+        ],
+        "responses": {
+          "200": {
+            "description": "Archived a reservation."
```

Comment:
what is the reponse? is there no response body?

Reply:
The response has no body..